src/Ganeti/HTools/Node.hs

```diff
@@ -133,11 +134,12 @@ type TagMap = Map.Map String Int
 data Node = Node
   { name     :: String    -- ^ The node name
   , alias    :: String    -- ^ The shortened name (for display purposes)
-  , tMem     :: Double    -- ^ Total memory (MiB)
-  , nMem     :: Int       -- ^ Node memory (MiB)
-  , fMem     :: Int       -- ^ Free memory (MiB)
+  , tMem     :: Double    -- ^ Total memory (MiB) (state-of-world)
```

Comment:
When would the total memory be different in state-of-world/state-of-record cases? A hardware failure or something like that?